detection_rules/mappings.py

```diff
@@ -73,3 +77,80 @@ def get_rta_files(self, rta_list=None, rule_ids=None):
                 rta_files.add(rta_path)
 
         return list(sorted(rta_files))
+
+
+def get_triggered_rules() -> dict:
+    """Get the rules that are triggered by each RTA."""
+    triggered_rules = {}
+    for rta_test in get_available_tests().values():
+        for rule_info in rta_test.get("siem", []):
+            rule_id = rule_info.get("rule_id")
+            for platform in rta_test.get("platforms", []):
```

Comment:
if no platform is defined, it implies compatibility with all, so that would erroneously create a gap here, right

Reply:
Platforms are required and specified when the `RtaMetadata` is created.  I've added a check for this as well.